### DISCUSSION TITLE
引用资源文件之后  项目启动报错

### DISCUSSION THREAD

**Discussion by liping94** [HandyOrg/HandyControl#1542] [General]
<ResourceDictionary.MergedDictionaries>
            <ResourceDictionary Source="pack://application:,,,/HandyControl;component/Themes/SkinDefault.xaml"/>
            <ResourceDictionary Source="pack://application:,,,/HandyControl;component/Themes/Theme.xaml"/>
        </ResourceDictionary.MergedDictionaries> 
![1710985336552](https://github.com/HandyOrg/HandyControl/assets/45651913/0713c197-68a9-4209-882b-57528dce633f)

**Comment by NaBian** [comment]
使用最新版本
